cfme/web_ui/__init__.py

```diff
@@ -144,8 +144,8 @@ class Region(Pretty):
     Note:
 
         When specifying a region title, omit the "Cloudforms Management Engine: " or "ManageIQ: "
-        prefix. They are included on every page, and different for the two versions of the
-        appliance, and :py:meth:`is_displayed` strips them off before checking for equality.
+        prefix. They're included on every page, and different for the two versions of the appliance,
+        and :py:meth:`is_displayed` strips them off before checking for equality.
```

Comment:
If you add a commit that reverts the changes and then squash it, it will disappear.